<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QMainWindow" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>习主席讲话强军篇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>习主席讲话党内篇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>习主席讲话改革篇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>习主席讲话强国篇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>习主席讲话外交篇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>习主席讲话西部篇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1.发展中国稳定中国的必由之路</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2.全体中华儿女的共同向往</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3.敢于啃硬骨头 敢于涉险滩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4.实现实实在在没有水分的增长</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5.充分发挥我国社会主义政治制度优越性</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6.让老百姓过上好日子</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>7.绿水青山就是金山银山</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>8.建设听党指挥能打胜仗作风优良的军队</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>9.建立新型国际关系</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10.打铁还需自身硬</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>请输入查找内容</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
